src/components/common/BottomNavigation.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import useModeStore from '../../stores/useModeStore';
+
+import { ReactComponent as ProposalIcon } from '../../assets/icons/BNB_proposal.svg';
+import { ReactComponent as FundingIcon } from '../../assets/icons/BNB_funding.svg';
+import { ReactComponent as RewardIcon } from '../../assets/icons/BNB_reward.svg';
+import { ReactComponent as MyIcon } from '../../assets/icons/BNB_my.svg';
+import { ReactComponent as RecommendIcon } from '../../assets/icons/BNB_rec.svg';
+
+const residentNavItems = [
+  { path: '/proposal', label: '제안', Icon: ProposalIcon },
+  { path: '/funding', label: '펀딩', Icon: FundingIcon },
+  { path: '/reward', label: '리워드', Icon: RewardIcon },
+  { path: '/my', label: '마이', Icon: MyIcon },
+];
+
+const founderNavItems = [
+  { path: '/proposal', label: '제안', Icon: ProposalIcon },
+  { path: '/recommend', label: '추천', Icon: RecommendIcon },
+  { path: '/funding', label: '펀딩', Icon: FundingIcon },
+  { path: '/my', label: '마이', Icon: MyIcon },
+];
+
+const BottomNavigation = () => {
+  const { userMode: mode } = useModeStore();
+  const location = useLocation();
+  const currentPath = location.pathname;
+
+  if (!mode) return null;
+  const navItems = mode === 'founder' ? founderNavItems : residentNavItems;
+
+  return (
+    <div
+      style={{
+        position: 'fixed',
+        bottom: 0,
+        left: '50%', // 1. 왼쪽 끝을 화면 중앙으로
+        transform: 'translateX(-50%)', // 2. 자신의 너비 절반만큼 왼쪽으로 이동
+        width: '390px', // 3. Figma 시안에 명시된 고정 너비
+        maxWidth: '100%',
```

Comment:
❗ Figma에서 기디분들이 390px을 기준으로 작업하고 있지만, breakpoint는 480px로 지정해 주셨습니다. 글로벌 스타일에서 width를 관리하고 있으므로 100%로만 지정해 주세요.
```suggestion
        width: '100%',
```